<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserProfile</class>
 <widget class="QWidget" name="UserProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WaveSight|Profile</string>
  </property>
  <widget class="QLabel" name="label_UserProfile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 75 bold 14pt &quot;Padauk&quot;;</string>
   </property>
   <property name="text">
    <string>User Profile</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>591</width>
     <height>421</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_UserProfileEmailID">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Email ID               </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_UserProfileContact">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>160</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Contact No</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_UserProfileContactText">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_UserProfileUID">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>40</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>User ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_UserProfileUIdText">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>root</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_UserProfileUName">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_UserProfileUNameText">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>spanidea</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_UserProfileUName_2">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>User Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_UserProfileUTypeText">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Administrator</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_UserProfileEmailIdText">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_UserProfilePasswordText">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_UserProfilePassword">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>190</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_UserProfileCPasswordText">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_UserProfileCPassword">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>220</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm Password   </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_EmailEdit">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>130</y>
      <width>45</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Sans Serif&quot;;

color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_EmailUpdate">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Sans Serif&quot;;
color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ContactEdit">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>160</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Sans Serif&quot;;

color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ContactUpdate">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Sans Serif&quot;;
color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PasswordEdit">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Sans Serif&quot;;
color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CPasswordUpdate">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>220</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Sans Serif&quot;;
color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
